finance.gemspec

```diff
@@ -17,7 +14,7 @@ SPEC = Gem::Specification.new do |s|
   s.add_dependency 'activesupport', '>= 3.2.0'
   s.add_dependency 'rake'
   s.add_development_dependency 'minitest', '>= 4.7.5'
-  s.files = FileList['README.md', 'COPYING', 'COPYING.LESSER', 'HISTORY', 'lib/**/*.rb', 'test/**/*.rb'].to_a
+  s.files = `git ls-files`.split("\n")
```

Comment:
I know your change isn't actually introducing this, but is it worth excluding specs? The "default" version of this can be found [here](https://github.com/hightower/heroku_pg_backups_archive/blob/master/heroku_pg_backups_archive.gemspec#L17)